riscv-page-tables/src/page_table.rs

```diff
@@ -879,6 +892,82 @@ impl<T: PagingMode> GuestStagePageTable<T> {
         self.do_invalidate_range(vaddr, len, false, pred)
     }
 
+    fn do_remap_range<F, P, M>(
+        &self,
+        paddr: P,
```

Comment:
Because Paddr could point anywhere, I'm not convinced this can be a safe function.

Reply:
Code no longer present.